### PR TITLE
add more comments for the timer in projectEventController

### PR DESCRIPTION
**Pull request by yangcao77**
Signed-off-by: Stephanie <stephanie.cao@ibm.com>

This PR is for issue #1998 
add more comments for the timer in projectEventController to help understand the code

### CODE REVIEW

**Comment by yangcao77** [comment]
@rajivnathan Can you help review this PR? Thanks

File: src/pfe/file-watcher/server/src/controllers/projectEventsController.ts [hunk 1]
```diff
@@ -252,15 +255,17 @@ export async function updateProjectForNewChange(projectID: string, timestamp: nu
                         changedFilesMap.delete(projectID);
                         logger.logProjectTrace("Removing project " + projectID + " from changedFilesMap.", projectID);
                     }
+                    // A build getting triggerred. Delete the project from the timerMap
```
Comment (rajivnathan) [inline src/pfe/file-watcher/server/src/controllers/projectEventsController.ts]: very minor nit: missing `is` in between `build` and `getting` also `triggerred` should be `triggered`.

File: src/pfe/file-watcher/server/src/controllers/projectEventsController.ts [hunk 2]
```diff
@@ -286,12 +291,14 @@ export async function updateProjectForNewChange(projectID: string, timestamp: nu
...
```
Comment (rajivnathan) [inline src/pfe/file-watcher/server/src/controllers/projectEventsController.ts]: nit: `triggered`

File: src/pfe/file-watcher/server/src/controllers/projectEventsController.ts [hunk 3]
```diff
@@ -286,12 +291,14 @@ export async function updateProjectForNewChange(projectID: string, timestamp: nu
...
                     timerMap.delete(projectID);
                     chunkRemainingMap.delete(projectID);
                 } catch (err) {
                     logger.logProjectError("Failed to set timeout for project update.", projectID);
                 }
             }, timeout);
+            // New chunk received, no build getting triggerred. Reset timer
```
Comment (rajivnathan) [inline src/pfe/file-watcher/server/src/controllers/projectEventsController.ts]: nit: triggered

File: src/pfe/file-watcher/server/src/controllers/projectEventsController.ts (hunk 1 above)
Comment (yangcao77) [reply src/pfe/file-watcher/server/src/controllers/projectEventsController.ts]: updated

File: src/pfe/file-watcher/server/src/controllers/projectEventsController.ts (hunk 2 above)
Comment (yangcao77) [reply src/pfe/file-watcher/server/src/controllers/projectEventsController.ts]: updated

File: src/pfe/file-watcher/server/src/controllers/projectEventsController.ts (hunk 3 above)
Comment (yangcao77) [reply src/pfe/file-watcher/server/src/controllers/projectEventsController.ts]: updated

**Pull request merged**